http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -310,9 +314,11 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       )
 
       // Create KSA
-      ksaName = KubernetesServiceAccount(s"${app.appName.value}-${config.galaxyAppConfig.serviceAccountSuffix}")
+      gsa = dbApp.app.googleServiceAccount
+      ksaName = ServiceAccountName(s"${app.appName.value}-${config.galaxyAppConfig.serviceAccountSuffix}")
```

Comment:
Sorry to cause conflicts but I change the KSA name in the PR I just merged. It's now a static string from reference.conf, we don't need to embed the appName.

Reply:
okay i'll work on rebasing now!